examples/stream.ru

```diff
@@ -8,17 +8,20 @@
 #   puma stream.ru                # gem install puma
 
 require 'sinatra/base'
+require 'sinatra/stream/reel'
 
 class Stream < Sinatra::Base
   get '/' do
-    content_type :txt
+    #content_type :txt
 
-    stream do |out|
-      out << "It's gonna be legen -\n"
+    #stream do |out|
+    Rack::StreamingResponse.new do |out|
+      out.hijack!(env)
```

Comment:
@TiagoCardoso1983 Hmm, good question, I think we shouldn't have to do this but I can't look into it now. Once I clear up some of the backlog I will take a look at new features.